### PR TITLE
Routes created, Navigation Module Ready wit Vuex

### PR DESCRIPTION
**Pull request by lakshyabatman**
Features:
1.  Routes are created, (/,/dashboard, /about , /settings/:id, /profile:id, /search )
2. Navigation Module is created.

- It has 2 phase, one when depending on where the user is logged in or not.

- [x] The phases are maintained by the use of prop: `isLogged: Boolean` _Which will be later migrated to Vuex_

- [x] Setting and profile route's: id in the navbar, is fetched from state `id:2` _Which will be later migrated to Vuex_
- [x]  Navigation Bar will be changed to sticky on integration with Vuex

### CODE REVIEW

**Comment by lakshyabatman** [comment]
@jaskirat2000 , @vaibhavdaren , @capedcrusader23 @devshiva619 , have a look on this.

**Comment by lakshyabatman** [comment]
@jaskirat2000 , I guess at this point. Navigation Module is done :) :fire:

**Comment by lakshyabatman** [comment]
Screenshot from 2019-05-26 14-41-16

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "social-platform-donut",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lakshyabatman/Social-Platform-Donut.git"
```
Comment (vaibhavdaren) [inline package.json]: ?

File: client/src/components/HelloWorld.vue [hunk 2]
```diff
@@ -1,59 +0,0 @@
-<template>
-  <div class="hello">
-    <h1>{{ msg }}</h1>
```
Comment (vaibhavdaren) [inline client/src/components/HelloWorld.vue]: remove this from commit

File: client/src/router.js [hunk 3]
```diff
@@ -10,16 +15,35 @@ export default new Router({
...
+      component: Settings
+    },
+    {
+      path: '/profile/:id',
+      name: 'ProfileView',
+      component: ProfileView
+    },
+    {
+      path: '/search',
+      name: 'Search', // Use $route.query to fetch queries
```
Comment (vaibhavdaren) [inline client/src/router.js]: nice comment :+1:

**Comment by vaibhavdaren** [comment]
nice progress :D

File: client/src/components/HelloWorld.vue (hunk 2 above)
Comment (lakshyabatman) [reply client/src/components/HelloWorld.vue]: Yes, It has been removed in next commit :D .

File: package.json (hunk 1 above)
Comment (lakshyabatman) [reply package.json]: I initiated the Git on vue project.I removed it. :)

**Comment by lakshyabatman** [comment]
Thankyou sir, I resolved the minor issues :)

**Pull request merged**